### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by YuliiaVishnia**
- [DEMO LINK](https://YuliiaVishnia.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://YuliiaVishnia.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,6 +9,22 @@
...
+    <header class="header">
+      <a href="#Home">
+        <img src="/src/images/logo.png" alt="logo" class="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#" class="nav__link is-active">Apple</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Samsung</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Smartphones</a></li>
+          <li class="nav__item"><a href="#" class="nav__link" data-qa="hover">Laptops &amp; Computers</a></li>
```
Comment (IvanFesenko) [inline src/index.html:21]:
```suggestion
          <li class="nav__item">
            <a 
              href="#"
              class="nav__link" 
              data-qa="hover"
            >
              Laptops &amp; Computers
            </a>
          </li>
```

File: src/index.html [hunk 2]
```diff
@@ -9,6 +9,22 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#Home">
+        <img src="/src/images/logo.png" alt="logo" class="logo">
```
Comment (IvanFesenko) [inline src/index.html:14]:
```suggestion
        <img 
          src="/src/images/logo.png" 
          alt="logo" 
          class="logo"
        >
```

**Review by IvanFesenko — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 3]
```diff
@@ -1 +1,100 @@
...
+  text-transform: uppercase;
+  color: #000;
+  text-decoration: none;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+a.is-active {
```
Comment (mykhalenych) [inline src/style.css:59]:
```suggestion
.is-active {
```

File: src/style.css [hunk 4]
```diff
@@ -1 +1,100 @@
...
+  position: relative;
+  color: #00acdc;
+}
+
+a.is-active::after {
```
Comment (mykhalenych) [inline src/style.css:64]:
```suggestion
.is-active::after {
```

File: src/style.css [hunk 5]
```diff
@@ -1 +1,100 @@
...
+  height: 60px;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+  text-decoration: none;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```
Comment (mykhalenych) [inline src/style.css:56]: create variable for color and use variable instead

**Review by mykhalenych — CHANGES_REQUESTED** [review]
your links aren't working

**Pull request closed without merging**